### DISCUSSION TITLE
Wrong piece movement issue for chessground chessboard

### DISCUSSION THREAD

**Discussion by coffeeDev98** [mliebelt/pgn-viewer#399] [Q&A]
Hey @mliebelt ,
Not sure as to the exact reason this issue occurs with chessground. The props passed are
```
    pgn: "",
    boardSize: `${boardfixedWidth}`,
    position: "start",
    startPlay: !analysis ? getStartPlay(pgn) : getStartPlay(roomSettingsPgn),
    manyGames: false,
    showCoords: true,
    orientation,
    ...(layout && { layout: layout }),
    showFen: true,
    coordsInner: false,
    headers: false,
    locale: "en",
    coordsFactor: "1.0",
    coordsFontSize: window.innerWidth > 800 ? "13px" : "",
    colorMarker: "",
    timerTime: "",
    hideMovesBefore: false,
    notation: "short",
    notationLayout: mobile ? "inline" : "list",
    theme: "brown",
    pieceStyle: "wikipedia",
  
```
Have you come across this issue? If so how did you resolve it...?

Video given below in reply to this discussion

**Comment by coffeeDev98** [comment]
https://user-images.githubusercontent.com/63295908/196845693-7a397271-61ab-477c-a83d-51035f5d7cfe.mov

**Reply by mliebelt** [reply]
Wow, I never saw something like that. I will try to reproduce it (first) with e.g. the configuration builder, and then will see. But I have no idea what could produce such a misbehavior.

**Reply by coffeeDev98** [reply]
Been trying to resolve this for a few days... unable to debug and find the cause of this. I had overwritten the CSS for width of ```${boardId}Inner``` to allow for some responsive layout... but apart from that, nothing much in terms of board CSS. 
Not sure whether that could be messing with the board logic. Ideally, CSS shouldn't be affecting this
